pynucastro/templates/amrexastro-cxx-microphysics/reaclib_rates.H.template

```diff
@@ -4,9 +4,12 @@
 #include <AMReX.H>
 #include <AMReX_Print.H>
 
+#include <tfactors.H>
 #include <actual_network.H>
+#include <partition_functions.H>
 
 using namespace Rates;
+using namespace Species;
```

Comment:
where the `Species` namespace is defined?

Reply:
`Species` is defined in `network_properties.H` which is automatically created at compile time by Microphysics